### PR TITLE
[PRO-1285] Documentation for the update project item cost endpoint

### PR DESCRIPTION
**Pull request by ketlane**
### Ticket
Link to ticket: https://teamleader.atlassian.net/browse/PRO-1285

### What has been done
Added documentation for the update project item's cost endpoint

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -2981,6 +2981,18 @@ Get a list of project items.
...
 
+### projectItems.update [POST /projectItems.update]
+
+Update a project item.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + id: `cfb4146d-06be-41f1-bb39-aa3c929c71dc` (string, required)
+        + unit_internal_cost: `123.30` (number, optional)
```
Comment (adognameddog) [inline apiary.apib]: Q: Shouldn't this be a money object? With an amount and a currency code?

Comment (ketlane) [reply apiary.apib]: Yes, fixed.

**Review by adognameddog — APPROVED** [review]

**Comment by adognameddog** [comment]
Added the hold label, since this functionality should not be made official just yet.

**Review by adognameddog — DISMISSED** [review]

**Review by ArnaudWeyts — DISMISSED** [review]

File: src/07-projects/project-items.apib [hunk 2]
```diff
@@ -71,4 +71,16 @@ Get a list of project items.
                 + updated_at: `2018-12-05T16:44:33+00:00` (string)
         + meta (object)
             + page (Pagination)
-            + matches: `120` (number)
\ No newline at end of file
+            + matches: `120` (number)
+
+### projectItems.updateCost [POST /projectItems.updateCost]
```
Comment (jenssegers) [inline src/07-projects/project-items.apib:115]: Q: Why not `.update`?

**Review by ArnaudWeyts — DISMISSED** [review]

File: src/07-projects/project-items.apib (hunk 2 above)
Comment (adognameddog) [reply src/07-projects/project-items.apib:115]: A: Because we only allow to update the cost.

**Review by adognameddog — DISMISSED** [review]

File: src/07-projects/project-items.apib (hunk 2 above)
Comment (jenssegers) [reply src/07-projects/project-items.apib:115]: If that field is the only thing that makes sense updating for a project item, in general, I think it's still an option. Will there be an update endpoint in the future? And if so, what will it include besides the cost?

I understand the reasoning to go for an explicit endpoint, but mainly thinking about consistency here. If we plan to add an `update` method in the future we can also already release that but with only the cost field for now.

If updating the cost has a bunch of side-effects and can't be done in combination with other properties being updated, then let's go for this endpoint. Just want to make sure it checks the boxes for being a separate endpoint.

**Review by adognameddog — APPROVED** [review]

**Review by ArnaudWeyts — APPROVED** [review]

**Review by joaoinacio — APPROVED** [review]

**Review by nickwaelkens — APPROVED** [review]

File: src/07-projects/project-items.apib (hunk 2 above)
Comment (ketlane) [reply src/07-projects/project-items.apib:115]: Updating the cost is quite specific as it also needs to update the purchase price of the originating order item, as a side-effect.

**Pull request merged**